benchmarks/probe_mxfp8_linear_narrow_output.py

```
@@ -0,0 +1,234 @@
+            milliseconds = float(start.elapsed_time(stop) / args.repeats)
+            results.append(
+                {
+                    "plan": plan.name,
+                    "finite": finite,
+                    "mismatches": mismatches,
+                    "max_abs_error": max_abs_error,
+                    "row_mismatches": row_mismatches,
+                    "milliseconds": milliseconds,
+                }
+            )
+        except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)